packages/analytics-js/src/services/StoreManager/types.ts

```diff
@@ -1,11 +1,27 @@
+import {
+  COOKIE_STORAGE,
+  LOCAL_STORAGE,
+  MEMORY_STORAGE,
+} from '@rudderstack/analytics-js-common/constants/storages';
 import {
   ICookieStorageOptions,
   IInMemoryStorageOptions,
   ILocalStorageOptions,
 } from '@rudderstack/analytics-js-common/types/Store';
+import {
+  CLIENT_DATA_STORE_COOKIE,
+  CLIENT_DATA_STORE_LS,
+  CLIENT_DATA_STORE_MEMORY,
+} from '@rudderstack/analytics-js/constants/storage';
 
 export type StoreManagerOptions = {
   cookieOptions?: Partial<ICookieStorageOptions>;
   localStorageOptions?: Partial<ILocalStorageOptions>;
   inMemoryStorageOptions?: Partial<IInMemoryStorageOptions>;
 };
+
+export const storageClientDataStoreNameMap = {
+  [COOKIE_STORAGE as string]: CLIENT_DATA_STORE_COOKIE,
```

Comment:
```suggestion
  [COOKIE_STORAGE]: CLIENT_DATA_STORE_COOKIE,
```

Reply:
When we will use this mapping dynamically it will not work. Either you have to define a type for the object keys or this way.

Reply:
I see. Weird.
You can declare those constants as strings instead.

Reply:
You can not declare a constant as a string.

Reply:
You can but eslint will show a warning that it's redundant.
Let's do this instead,
```
export const storageClientDataStoreNameMap: Record<string, string> = {
  [COOKIE_STORAGE]: CLIENT_DATA_STORE_COOKIE,
  [LOCAL_STORAGE]: CLIENT_DATA_STORE_LS,
  [MEMORY_STORAGE]: CLIENT_DATA_STORE_MEMORY,
};
```

Reply:
Done